### PR TITLE
use pinned images

### PR DESCRIPTION
**Pull request by cpmeadors**
Image urls need to be pinned with shas.  This will allow konflux to update the shas as new images are built.

I also added an annotation to the ocp bundle to define the subscriptions.

### CODE REVIEW

File: bundle-ocp/manifests/instaslice-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Drivers and plugins
-    containerImage: registry.redhat.io/dynamic-accelerator-slicer-tech-preview/instaslice-operator-rhel9:0.1.0
+    containerImage: registry.redhat.io/dynamic-accelerator-slicer-tech-preview/instaslice-operator-rhel9@sha256:0d8fcbe0a52b59cea9f699a4bccea9e4c148a457447a76602a68eabda2d17184
```
Comment (sairameshv) [inline bundle-ocp/manifests/instaslice-operator.clusterserviceversion.yaml:25]: Do we have to update these every time or is it a one time job to add/pin shas?

Comment (cpmeadors) [reply bundle-ocp/manifests/instaslice-operator.clusterserviceversion.yaml:25]: konflux is supposed to update these references with "nudges" (how relationships are defined in konflux)  Should be a one time change assuming I get it right.

Comment (sairameshv) [reply bundle-ocp/manifests/instaslice-operator.clusterserviceversion.yaml:25]: Okay
/lgtm
/approve

**Review by sairameshv — APPROVED** [review]

**Pull request merged**